### ISSUE TITLE
If Else template construct not discussed in docs

### ISSUE THREAD

**Issue by Xliff** [croservices/cro-webapp#79]
Is there a repository for them that I can make a PR against?

**Comment by Altai-man** [comment]
Yes, see https://github.com/croservices/cro/blob/master/docs/reference/cro-webapp-template.md

There is no if-else syntax though, so it has to be written as if + !if pair.

**Comment by Xliff** [comment]
Actually, there is. It should have been a part of the most recent release. See:

https://github.com/croservices/cro-webapp/blob/master/t/test-data/cond-if-else.crotmp

You are correct in that there are two sections, but the else part does not need a conditional.

**Comment by jnthn** [comment]
> It should have been a part of the most recent release. 

It hasn't appeared in a release yet (last Cro release was long ago). Updating docs to cover new additions since the last release is one of the main release blockers, so doc additions are welcome to help things along.
